### PR TITLE
Remotely Entertaining

### PR DESCRIPTION
**Pull request by christinamattinson**
This blog is a follow on from the keynote session we ran on remote
working, please provide feedback and any other key points you think
would be useful.

### CODE REVIEW

File: _posts/2016-06-10-Remotely-Entertaining.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: 'Remote'ly Entertaining
+categories: Remote Working
+author: christina_mattinson
+tags: remote working
+comments: true
+excerpt: Remote working is not that scary, lots of useful tips and guidance on how we do it at HX!
+
+So our Web team has expanded to Sofia in Bulgaria, with developers joining our already established Scrum teams. With this change to our team dynamics we have learnt how to work remotely really well and of course we still have more areas for improvements. Last week we delivered our ambitious keynote on remote working at #HXConf which definately highlighted some of the challenges . . .this was intentional ;)
```
Comment (hpoom) [inline _posts/2016-06-10-Remotely-Entertaining.md]: I would change "Scrum teams" to "agile teams"

File: _posts/2016-06-10-Remotely-Entertaining.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+- Engagement is really important so try to make the remote workers feel like they are not at a disadvantage, use multiple cameras to show what is happening in the room, it is much easier to follow a conversation if you can see body language. 
+- Think about creative ways to collaborate - we have had some great tools and electronic boards introduced to share ideas on.
+- Oh and don’t forget screen share on Google Hangouts.
+- Think about how best to use the equipment, desk microphones have a limited range so think about the how you are projecting your voice so you can be heard clearly. 
+- Additionally have you ever been in a meeting where there are two conversations happening . . . it is so hard to follow. This is multiplied when listening in one a remote call.
+
+**Oh and one final point, our web team now we have two different types of remote meetings:**
+- An already established team starting to work remotely.
+- A new member of the team working remotely with new people, this introduces new challenges because the team familiarity is not yet established, making that individual feel part of the team sooner will create a more effective team!
```
Comment (hpoom) [inline _posts/2016-06-10-Remotely-Entertaining.md]: Full stop before "Making" 

`is not yet established. Making that individual`

**Comment by hpoom** [comment]
Couple of minor comments, but I am no grammar or spelling expert. A 👍 from me.

File: _posts/2016-06-10-Remotely-Entertaining.md (hunk 1 above)
Comment (pmcnr-hx) [inline _posts/2016-06-10-Remotely-Entertaining.md]: `,` after `dynamics`.

File: _posts/2016-06-10-Remotely-Entertaining.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+author: christina_mattinson
+tags: remote working
+comments: true
+excerpt: Remote working is not that scary, lots of useful tips and guidance on how we do it at HX!
+
+So our Web team has expanded to Sofia in Bulgaria, with developers joining our already established Scrum teams. With this change to our team dynamics we have learnt how to work remotely really well and of course we still have more areas for improvements. Last week we delivered our ambitious keynote on remote working at #HXConf which definately highlighted some of the challenges . . .this was intentional ;) 
+
+**This blog is a follow up from that to highlight the key points made during the session and some more.**
+
+Organising a call can be extremely simple as long as everyone is on the same page, as a business we use google calendar which organises our calls for us.
```
Comment (pmcnr-hx) [inline _posts/2016-06-10-Remotely-Entertaining.md]: Two separate sentences here? I.e `.` instead of the `,`?

File: _posts/2016-06-10-Remotely-Entertaining.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+So our Web team has expanded to Sofia in Bulgaria, with developers joining our already established Scrum teams. With this change to our team dynamics we have learnt how to work remotely really well and of course we still have more areas for improvements. Last week we delivered our ambitious keynote on remote working at #HXConf which definately highlighted some of the challenges . . .this was intentional ;) 
+
+**This blog is a follow up from that to highlight the key points made during the session and some more.**
+
+Organising a call can be extremely simple as long as everyone is on the same page, as a business we use google calendar which organises our calls for us. 
+### The benefits
+- Add your invites to check availability, they get a notification in advance. 
+- Google hangouts create a shared accessible link to host the call.
+- The green bars in the right hand corner do a sound check for you, thanks Google!
```
Comment (pmcnr-hx) [inline _posts/2016-06-10-Remotely-Entertaining.md]: `:` instead of `,`?

File: _posts/2016-06-10-Remotely-Entertaining.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+### Setting up your equipment
+- Use good equipment, desk speakers and microphones if in a group.
+- Individuals calling in, use good headsets that restrict background noise (our IT department issue some really good quality headsets).
+- If you do not have the equipment you need get it ordered! (James Penning is our go to guy for this!).
```
Comment (pmcnr-hx) [inline _posts/2016-06-10-Remotely-Entertaining.md]: Does it make sense to name James for the wider audience?

File: _posts/2016-06-10-Remotely-Entertaining.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+- Think about your environment, you don’t want to have a remote call in a loud office where you will distract others and the background noise will distract the call.
```
Comment (pmcnr-hx) [inline _posts/2016-06-10-Remotely-Entertaining.md]: `.` instead of `,`.

File: _posts/2016-06-10-Remotely-Entertaining.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+- Make sure your equipment is set up properly, two places to check your google call settings and the default setup of your machine - change both if you are not sure.
+- The quality of the call will depend on the quality of the internet connection - use ethernet or strong WIFI connections - if one connection on the call has a bad connection it will impact everyones call.
+- Sometimes if the microphone and speaker are two close you can get an extremely distracting echo of your own voice, and we all hate the sound of our own voice!
```
Comment (pmcnr-hx) [inline _posts/2016-06-10-Remotely-Entertaining.md]: `two` → `too`.

File: _posts/2016-06-10-Remotely-Entertaining.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+- Be organised with the set up otherwise it can cut into your meeting time.
+
+### How to put you remote team member first
+- Actively ask questions and encourage engagement. 
+- If you are the remote worker speak up, let the team know if you are finding the call challenging for whatever reason.
+- Think about time zones, try to organise meetings at times that are suitable for everyone and try to spread out meetings.
```
Comment (pmcnr-hx) [inline _posts/2016-06-10-Remotely-Entertaining.md]: `.` instead of `,`.

File: _posts/2016-06-10-Remotely-Entertaining.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+- Make sure your equipment is set up properly, two places to check your google call settings and the default setup of your machine - change both if you are not sure.
+- The quality of the call will depend on the quality of the internet connection - use ethernet or strong WIFI connections - if one connection on the call has a bad connection it will impact everyones call.
+- Sometimes if the microphone and speaker are two close you can get an extremely distracting echo of your own voice, and we all hate the sound of our own voice!
+- Be organised with the set up otherwise it can cut into your meeting time.
+
+### How to put you remote team member first
+- Actively ask questions and encourage engagement. 
+- If you are the remote worker speak up, let the team know if you are finding the call challenging for whatever reason.
+- Think about time zones, try to organise meetings at times that are suitable for everyone and try to spread out meetings. 
+- You tend to use up more energy following conversations remote so don’t organise to many closely together!
```
Comment (pmcnr-hx) [inline _posts/2016-06-10-Remotely-Entertaining.md]: `to` → `too`.

File: _posts/2016-06-10-Remotely-Entertaining.md [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+- Engagement is really important so try to make the remote workers feel like they are not at a disadvantage, use multiple cameras to show what is happening in the room, it is much easier to follow a conversation if you can see body language.
```
Comment (pmcnr-hx) [inline _posts/2016-06-10-Remotely-Entertaining.md]: `, it is much` → `. It is much`

**Comment by pmcnr-hx** [comment]
:smile_cat: nice! Mostly grammar stuff. I'll have another read after other reviewers have fed back.

File: _posts/2016-06-10-Remotely-Entertaining.md [hunk 11]
```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: 'Remote'ly Entertaining
+categories: Remote Working
+author: christina_mattinson
+tags: remote working
+comments: true
+excerpt: Remote working is not that scary, lots of useful tips and guidance on how we do it at HX!
+
+So our Web team has expanded to Sofia in Bulgaria, with developers joining our already established agile teams. With this change to our team dynamics, we have learnt how to work remotely really well and of course we still have more areas for improvements. Last week we delivered our ambitious keynote on remote working at #HXConf which definately highlighted some of the challenges . . .this was intentional ;)
```
Comment (steve-at-hx) [inline _posts/2016-06-10-Remotely-Entertaining.md]: > we have learnt how to work remotely really well and of course we still have more areas for improvements
Could this read:
> we have learnt how to work remotely really well, while we still identify new areas for improvements as our team grows and the remote tech around us improves.

File: _posts/2016-06-10-Remotely-Entertaining.md [hunk 12]
```diff
@@ -0,0 +1,43 @@
...
+author: christina_mattinson
+tags: remote working
+comments: true
+excerpt: Remote working is not that scary, lots of useful tips and guidance on how we do it at HX!
+
+So our Web team has expanded to Sofia in Bulgaria, with developers joining our already established agile teams. With this change to our team dynamics, we have learnt how to work remotely really well and of course we still have more areas for improvements. Last week we delivered our ambitious keynote on remote working at #HXConf which definately highlighted some of the challenges . . .this was intentional ;) 
+
+**This blog is a follow up from that to highlight the key points made during the session and some more.**
+
+Organising a call can be extremely simple as long as everyone is on the same page. As a business we use google calendar which organises our calls for us.
```
Comment (steve-at-hx) [inline _posts/2016-06-10-Remotely-Entertaining.md]: As you've mentioned Google Calendar, I think you either need to make this a section on setting up the call or also reference Hangouts in the intro.

File: _posts/2016-06-10-Remotely-Entertaining.md [hunk 13]
```diff
@@ -0,0 +1,43 @@
...
+tags: remote working
+comments: true
+excerpt: Remote working is not that scary, lots of useful tips and guidance on how we do it at HX!
+
+So our Web team has expanded to Sofia in Bulgaria, with developers joining our already established agile teams. With this change to our team dynamics, we have learnt how to work remotely really well and of course we still have more areas for improvements. Last week we delivered our ambitious keynote on remote working at #HXConf which definately highlighted some of the challenges . . .this was intentional ;) 
+
+**This blog is a follow up from that to highlight the key points made during the session and some more.**
+
+Organising a call can be extremely simple as long as everyone is on the same page. As a business we use google calendar which organises our calls for us. 
+### The benefits
```
Comment (steve-at-hx) [inline _posts/2016-06-10-Remotely-Entertaining.md]: Might make this an easier read if you could add a little intro to each section before the bullets?

File: _posts/2016-06-10-Remotely-Entertaining.md [hunk 14]
```diff
@@ -0,0 +1,43 @@
...
+- Use good equipment, desk speakers and microphones if in a group.
+- Individuals calling in, use good headsets that restrict background noise (our IT department issue some really good quality headsets).
+- If you do not have the equipment you need get it ordered!
+- Think about your environment, you don’t want to have a remote call in a loud office where you will distract others and the background noise will distract the call.
+- Make sure your equipment is set up properly, two places to check your google call settings and the default setup of your machine - change both if you are not sure.
+- The quality of the call will depend on the quality of the internet connection - use ethernet or strong WIFI connections - if one connection on the call has a bad connection it will impact everyones call.
+- Sometimes if the microphone and speaker are too close you can get an extremely distracting echo of your own voice, and we all hate the sound of our own voice!
+- Be organised with the set up otherwise it can cut into your meeting time.
+
+### How to put you remote team member first
```
Comment (steve-at-hx) [inline _posts/2016-06-10-Remotely-Entertaining.md]: Building on comment above. It would be great to summarise in a few words what the risks are around remote members, i.e. them not feeling part of the conversation.

File: _posts/2016-06-10-Remotely-Entertaining.md [hunk 15]
```diff
@@ -0,0 +1,43 @@
...
+- Think about time zones. Try to organise meetings at times that are suitable for everyone and try to spread out meetings. 
+- You tend too use up more energy following conversations remote so don’t organise to many closely together!
+- Engagement is really important so try to make the remote workers feel like they are not at a disadvantage, use multiple cameras to show what is happening in the room. It is much easier to follow a conversation if you can see body language. 
+- Think about creative ways to collaborate - we have had some great tools and electronic boards introduced to share ideas on.
+- Oh and don’t forget screen share on Google Hangouts.
+- Think about how best to use the equipment, desk microphones have a limited range so think about the how you are projecting your voice so you can be heard clearly. 
+- Additionally have you ever been in a meeting where there are two conversations happening . . . it is so hard to follow. This is multiplied when listening in one a remote call.
+
+**Oh and one final point, our web team now have two different types of remote meetings:**
```
Comment (steve-at-hx) [inline _posts/2016-06-10-Remotely-Entertaining.md]: Could you maybe abandon the bullet point approach here and close on a bit more detail about the varying needs of different groups and the importance being on the consideration of these needs over a prescribed approach, using these two groups as an example? I think the way you guys had to mitigate the problems in your presentation is a good example of this! You made it work because you were aware and considerate of the issues.

File: _posts/2016-06-10-Remotely-Entertaining.md [hunk 16]
```diff
@@ -7,37 +7,40 @@ tags: remote working
 comments: true
 excerpt: Remote working is not that scary, lots of useful tips and guidance on how we do it at HX!
 
-So our Web team has expanded to Sofia in Bulgaria, with developers joining our already established agile teams. With this change to our team dynamics, we have learnt how to work remotely really well and of course we still have more areas for improvements. Last week we delivered our ambitious keynote on remote working at #HXConf which definately highlighted some of the challenges . . .this was intentional ;) 
+So our Web team has expanded to Sofia in Bulgaria, with developers joining our already established agile teams. With this change to our team dynamics, we have learnt how to work remotely really well, while we still identify new areas for improvements as our team grows and the remote tech around us improves. Last week we delivered our ambitious keynote on remote working at #HXConf which definitely highlighted some of the challenges . . . this was intentional ;)
```
Comment (pmcnr-hx) [inline _posts/2016-06-10-Remotely-Entertaining.md]: _while we are still identifying_ seems to read better, no?
Link `#HXConf` to `https://twitter.com/hashtag/hxconf` for context?
Real emoji → 😉?

File: _posts/2016-06-10-Remotely-Entertaining.md [hunk 17]
```diff
@@ -7,37 +7,40 @@ tags: remote working
...
 
-**This blog is a follow up from that to highlight the key points made during the session and some more.**
+This blog is a follow up from that to highlight the key points made during the session from starting a call, to facilitating an effective remote session and some more.
 
-Organising a call can be extremely simple as long as everyone is on the same page. As a business we use google calendar which organises our calls for us. 
+Organising a call can be extremely simple as long as everyone is on the same page. As a business we use google calendar to organise the time and attendees on the call. Google hangouts is the tool we use to host the calls as the functionality is already there (and it is free).
 ### The benefits
 - Add your invites to check availability, they get a notification in advance. 
-- Google hangouts create a shared accessible link to host the call.
+- Google hangouts create a shared accessible link to host the call, accessible to anyone with a gmail account.
```
Comment (pmcnr-hx) [inline _posts/2016-06-10-Remotely-Entertaining.md]: `create` → `creates`

File: _posts/2016-06-10-Remotely-Entertaining.md [hunk 18]
```diff
@@ -7,37 +7,40 @@ tags: remote working
...
 - The green bars in the right hand corner do a sound check for you: thanks Google!
 
+Over the last few weeks we have been exposed to remote planning meetings, stand ups,  catch ups and retrospectives. Each different type of meetings introduced new challenges and we tested out various sessions in different environments with different equipment. Now looking back we have learnt some really useful tips.
```
Comment (pmcnr-hx) [inline _posts/2016-06-10-Remotely-Entertaining.md]: Shouldn't the introductory paragraph be in the next section?

File: _posts/2016-06-10-Remotely-Entertaining.md [hunk 19]
```diff
@@ -7,37 +7,40 @@ tags: remote working
...
 - Sometimes if the microphone and speaker are too close you can get an extremely distracting echo of your own voice, and we all hate the sound of our own voice!
 - Be organised with the set up otherwise it can cut into your meeting time.
 
+Another really important point when facilitating a remote meeting is to put your remote worker first as their contribution is really important and sometimes they need some encouragement to speak up.
 ### How to put you remote team member first
 - Actively ask questions and encourage engagement. 
 - If you are the remote worker speak up, let the team know if you are finding the call challenging for whatever reason.
 - Think about time zones. Try to organise meetings at times that are suitable for everyone and try to spread out meetings. 
-- You tend too use up more energy following conversations remote so don’t organise to many closely together!
+- You tend too use up more energy following conversations remotely so don’t organise to many closely together!
```
Comment (pmcnr-hx) [inline _posts/2016-06-10-Remotely-Entertaining.md]: `tend too` → `tend to`

File: _posts/2016-06-10-Remotely-Entertaining.md [hunk 20]
```diff
@@ -7,37 +7,40 @@ tags: remote working
...
\ No newline at end of file
+Oh and one final point, our web team now have two different types of remote meetings.
+An already established agile team starting to work remotely and a new member of the team working remotely with new people. The second scenario introduces new challenges because the team familiarity is not yet established. Making that individual feel part of the team sooner will create a more effective team, better collaboration and more open and honest relationships! 
+
+Remote working is not more difficult, it is just different. Team habits need to change slightly but once adopted you will see the benefits and contributions from your remote team member, no matter where they are based. One useful tool my team recently adopted was running a remote themed retrospective after a few weeks of practice. This allowed us to highlight what we were doing really well so we could refer back to our points if our habits slip back. It also enabled the team to highlight even further advances we could make which I will share. We realised that we had challenges with our equipment and testing mobile bugs remotely was difficult due to access to mobile devices.
```
Comment (pmcnr-hx) [inline _posts/2016-06-10-Remotely-Entertaining.md]: _which I will share in a future post_?
Also the last sentence reads like a rather abrupt ending, it feels like you should have an additional sentence to conclude the piece, or rework the final two sentences.

**Comment by carolinewoodhams** [comment]
Hi @christinamattinson - have you reviewed all the comments yet? You've got +1 from Si, Pedro's made a few comments. be good to get this post merged before the weekend,  if you can take a look at this today? :)

**Comment by christinamattinson** [comment]
Sorry for delay, I think this is ready to merge now @pmcnr-hx can you give it a final review as you had a fair amount of feedback. 

Thanks

File: _posts/2016-06-10-Remotely-Entertaining.md [hunk 21]
```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: 'Remote'ly Entertaining
+categories: Remote Working
+author: christina_mattinson
+tags: remote working
+comments: true
+excerpt: Remote working is not that scary, lots of useful tips and guidance on how we do it at HX!
+
+So our Web team has expanded to Sofia in Bulgaria, with developers joining our already established agile teams. With this change to our team dynamics, we have learnt how to work remotely really well, while we are still identifying new areas for improvements as our team grows and the remote tech around us improves. Last week we delivered our ambitious keynote on remote working at [#HXConf](https://twitter.com/hashtag/hxconf) which definitely highlighted some of the challenges . . . this was intentional ;)
```
Comment (pmcnr-hx) [inline _posts/2016-06-10-Remotely-Entertaining.md]: Emoji instead for the extra punch? 😉

File: _posts/2016-06-10-Remotely-Entertaining.md [hunk 22]
```diff
@@ -0,0 +1,46 @@
...
+
+This blog is a follow up from that to highlight the key points made during the session from starting a call, to facilitating an effective remote session and some more.
+
+Organising a call can be extremely simple as long as everyone is on the same page. As a business we use google calendar to organise the time and attendees on the call. Google hangouts is the tool we use to host the calls as the functionality is already there (and it is free).
```
Comment (pmcnr-hx) [inline _posts/2016-06-10-Remotely-Entertaining.md]: Be consistent and always capitalise *Google* (look for other instances below).

File: _posts/2016-06-10-Remotely-Entertaining.md [hunk 23]
```diff
@@ -0,0 +1,46 @@
...
+### The benefits
+- Add your invites to check availability, they get a notification in advance.
+- Google hangouts creates a shared accessible link to host the call, accessible to anyone with a gmail account.
+- The green bars in the right hand corner do a sound check for you: thanks Google!
+
+Over the last few weeks we have been exposed to remote planning meetings, stand ups,  catch ups and retrospectives. Each different type of meetings introduced new challenges and we tested out various sessions in different environments with different equipment. Now looking back we have learnt some really useful tips.
```
Comment (pmcnr-hx) [inline _posts/2016-06-10-Remotely-Entertaining.md]: _stand-ups_ and _catch-ups_ should be hyphened.

File: _posts/2016-06-10-Remotely-Entertaining.md [hunk 24]
```diff
@@ -0,0 +1,46 @@
...
+### Setting up your equipment
+- Use good equipment, desk speakers and microphones if in a group.
+- Individuals calling in, use good headsets that restrict background noise.
+- If you do not have the equipment you need get it ordered!
```
Comment (pmcnr-hx) [inline _posts/2016-06-10-Remotely-Entertaining.md]: _need *to* get it ordered_

File: _posts/2016-06-10-Remotely-Entertaining.md [hunk 25]
```diff
@@ -0,0 +1,46 @@
...
+Organising a call can be extremely simple as long as everyone is on the same page. As a business we use google calendar to organise the time and attendees on the call. Google hangouts is the tool we use to host the calls as the functionality is already there (and it is free).
+### The benefits
+- Add your invites to check availability, they get a notification in advance.
+- Google hangouts creates a shared accessible link to host the call, accessible to anyone with a gmail account.
+- The green bars in the right hand corner do a sound check for you: thanks Google!
+
+Over the last few weeks we have been exposed to remote planning meetings, stand ups,  catch ups and retrospectives. Each different type of meetings introduced new challenges and we tested out various sessions in different environments with different equipment. Now looking back we have learnt some really useful tips.
+### Setting up your equipment
+- Use good equipment, desk speakers and microphones if in a group.
+- Individuals calling in, use good headsets that restrict background noise.
```
Comment (pmcnr-hx) [inline _posts/2016-06-10-Remotely-Entertaining.md]: **:** rather than **,** here?

File: _posts/2016-06-10-Remotely-Entertaining.md [hunk 26]
```diff
@@ -0,0 +1,46 @@
...
+- If you do not have the equipment you need get it ordered!
+- Think about your environment, you don’t want to have a remote call in a loud office where you will distract others and the background noise will distract the call.
```
Comment (pmcnr-hx) [inline _posts/2016-06-10-Remotely-Entertaining.md]: _Think about your environment. You..._

File: _posts/2016-06-10-Remotely-Entertaining.md [hunk 27]
```diff
@@ -0,0 +1,46 @@
...
+- Make sure your equipment is set up properly, two places to check your google call settings and the default setup of your machine - change both if you are not sure.
```
Comment (pmcnr-hx) [inline _posts/2016-06-10-Remotely-Entertaining.md]: _Make sure your equipment is set up properly. Two places to check:_

File: _posts/2016-06-10-Remotely-Entertaining.md [hunk 28]
```diff
@@ -0,0 +1,46 @@
...
+- The quality of the call will depend on the quality of the internet connection - use ethernet or strong WIFI connections - if one connection on the call has a bad connection it will impact everyones call.
```
Comment (pmcnr-hx) [inline _posts/2016-06-10-Remotely-Entertaining.md]: I'd just use full stops here to separate the sentences.

File: _posts/2016-06-10-Remotely-Entertaining.md [hunk 29]
```diff
@@ -0,0 +1,46 @@
...
+- Sometimes if the microphone and speaker are too close you can get an extremely distracting echo of your own voice, and we all hate the sound of our own voice!
+- Be organised with the set up otherwise it can cut into your meeting time.
+
+Another really important point when facilitating a remote meeting is to put your remote worker first as their contribution is really important and sometimes they need some encouragement to speak up.
+### How to put you remote team member first
+- Actively ask questions and encourage engagement.
+- If you are the remote worker speak up, let the team know if you are finding the call challenging for whatever reason.
+- Think about time zones. Try to organise meetings at times that are suitable for everyone and try to spread out meetings.
+- You tend to use up more energy following conversations remotely so don’t organise to many closely together!
```
Comment (pmcnr-hx) [inline _posts/2016-06-10-Remotely-Entertaining.md]: to many → too many

File: _posts/2016-06-10-Remotely-Entertaining.md [hunk 30]
```diff
@@ -0,0 +1,46 @@
...
+- Engagement is really important so try to make the remote workers feel like they are not at a disadvantage, use multiple cameras to show what is happening in the room. It is much easier to follow a conversation if you can see body language.
```
Comment (pmcnr-hx) [inline _posts/2016-06-10-Remotely-Entertaining.md]: **:** instead of **,**?

File: _posts/2016-06-10-Remotely-Entertaining.md [hunk 31]
```diff
@@ -0,0 +1,46 @@
...
+- Think about creative ways to collaborate - we have had some great tools and electronic boards introduced to share ideas on.
+- Oh and don’t forget screen share on Google Hangouts.
+- Think about how best to use the equipment, desk microphones have a limited range so think about the how you are projecting your voice so you can be heard clearly.
```
Comment (pmcnr-hx) [inline _posts/2016-06-10-Remotely-Entertaining.md]: Full stop instead of the comma?

File: _posts/2016-06-10-Remotely-Entertaining.md [hunk 32]
```diff
@@ -0,0 +1,46 @@
...
+- Additionally have you ever been in a meeting where there are two conversations happening . . . it is so hard to follow. This is multiplied when listening in on a remote call.
+
+Oh and one final point, our web team now have two different types of remote meetings.
```
Comment (pmcnr-hx) [inline _posts/2016-06-10-Remotely-Entertaining.md]: I would remove the *Oh and* here and just start with *One final*.

**Comment by pmcnr-hx** [comment]
Just a few (mostly pedantic) comments from me, but it's good to go: 👍 (no need for further reviews from me).
